index.html

```diff
@@ -52,5 +52,149 @@ <h1>Hey there. I'm <br>Victor<br><span>
         </div>
         
     </section>
+    <section class="work">
+        <div class="work-title">
+            <h2>My Recent <br>Works</h2>
+            <img src="images/Indicator.png" alt="indicator line">
```

Comment:
- [ ] Great working on this section as requested by the previous reviewer. However, in smaller screen sizes, the `About Me` is overlapping the paragraph below it. Please fix that. 
- See below:
![image](https://user-images.githubusercontent.com/52443674/159116537-6dab3ee3-5c1d-498e-857c-7549153d04ca.png)

Reply:
@karvel-code  Thanks Elvis, I have adjusted the font-size and padding. Can you please quickly have a look before I submit it again